UnityProject/Assets/Mirror/Runtime/Transports/Ignorance/Editor/Toolbox.cs

```diff
@@ -22,19 +23,19 @@ public static void RevealEnetLibraryName()
         [MenuItem("Ignorance/RTFM/Github Repo")]
         private static void LaunchGithubRepo()
         {
-            UnityEngine.Application.OpenURL("https://github.com/SoftwareGuy/Ignorance");
+	        Debug.LogError("https://github.com/SoftwareGuy/Ignorance");
         }
 
         [MenuItem("Ignorance/RTFM/Issue Tracker")]
         private static void LaunchGithubIssueTracker()
         {
-            UnityEngine.Application.OpenURL("https://github.com/SoftwareGuy/Ignorance/issues");
+              Debug.LogError("https://github.com/SoftwareGuy/Ignorance/issues");
         }
 
         [MenuItem("Ignorance/RTFM/ENet-C# Repo")]
         private static void LaunchENetCSharpForkRepo()
         {
-            UnityEngine.Application.OpenURL("https://github.com/SoftwareGuy/ENet-CSharp");
+            Debug.LogError("https://github.com/SoftwareGuy/ENet-CSharp");
```

Comment:
this doesn't sound like a good solution imo

Reply:
idk, this just needed for the editor not sure if it's going to be in the actual ignorance assembly or in the main assets one,